firmware/hw_layer/adc/adc_inputs.cpp

```diff
@@ -74,7 +74,7 @@ AdcDevice::AdcDevice(ADCConversionGroup* hwConfig, adcsample_t *buf, size_t buf_
 	hwConfig->sqr5 = 0;
 #endif /* ADC_MAX_CHANNELS_COUNT */
 	memset(hardwareIndexByIndernalAdcIndex, EFI_ADC_NONE, sizeof(hardwareIndexByIndernalAdcIndex));
-	memset(internalAdcIndexByHardwareIndex, 0xFFFFFFFF, sizeof(internalAdcIndexByHardwareIndex));
+	memset(internalAdcIndexByHardwareIndex, 0xFF, sizeof(internalAdcIndexByHardwareIndex));
```

Comment:
not to mention that memset actually:
`The value is passed as an int, but the function fills the block of memory using the unsigned char conversion of this value.`

Reply:
So it should have been 0xff all along 🙂

Reply:
It just a bit of cosmetic.